README.CentOS6.md

```diff
@@ -14,39 +14,97 @@ For further details on the install process, please see the LORIS GitHub Wiki Cen
 
 Default dependencies installed by CentOS 6.x may not meet the version requirements LORIS deployment or development.
 * MySQL 5.7 is supported for LORIS 17.*
-* PHP 7 is supported for LORIS 17.* - upgrade your PHP manually
+* PHP 7 is supported for LORIS 17.* - see below to upgrade your PHP manually
 
 The yum packages to be installed vary from any Ubuntu packages referenced in the LORIS README.
 
 The following should be installed with yum:
- 
-```bash
-yum install httpd
-yum install php
-yum install php-pdo
-yum install php-mysql
-yum install mysql
+ * Apache2
+ * PHP
+ * MySQL
+ * PHP Composer
+
+**Apache2:**
+```
+sudo yum install httpd 
+sudo service httpd start
+```
+**PHP:**
+```
+sudo yum install php php-pdo php-mysql 
 ```
 
-PHP Composer must also be installed:
-```bash
-curl -sS https://getcomposer.org/installer | php
+*NOTE:* As of Loris 18.0 php7 is required (but not yet officially supported by CentOS). To upgrade php follow the following instructions
+``` 
+# update php5 -> php7
+curl 'https://setup.ius.io/' -o setup-ius.sh
+sudo bash setup-ius.sh
+
+# update php7 specific packages
+sudo yum remove php-cli mod_php php-common
+sudo yum install php70u-json php70-xml mod_php70u php70u-cli php70u-mysqlnd php70u-mbstring
+``` 
+**MySQL:**
+
+*Note:* Loris developers (those NOT working with a .zip release codebase) should skip steps relating to hosting mysql locally. Contact sysadmins for database credentials directly.
+``` 
+sudo yum install mysql mysql-server
+```
+The above command will download MariaDB by default in CentOS7 [(see paragraph2)](https://www.digitalocean.com/community/tutorials/how-to-install-mysql-on-centos-7).
+
+Check what version you have installed by running:
+```
+mysql -v
+```
+If you have MySQL:
+```
+sudo service mysqld start
+sudo service mysqld status
+```
+If you have MariaDB:
+```
+sudo service mariadb start
+sudo service mariadb status
+```
+The two versions should act nearly identically in all other respects. 
+
+To finalise the MySQL/MariaDB installation: 
+```
+mysql_secure_installation
+```
+(follow instructions to create a password the root user):
+
+**PHP Composer:**
+```
+sudo curl -sS https://getcomposer.org/installer | php
 sudo mv composer.phar /usr/local/bin/composer
 ```
+# LORIS code base
 
-If you just want to get Loris running and upgrade your PHP later, run composer with the `--no-dev` option. 
+Download the latest release from the [releases page](https://github.com/aces/Loris/releases) to the home directory (~/), unzip it, and copy the contents to the project directory, `/var/www/loris` (we recommend naming your project directory `loris`, although you can use a different naming convention). 
+```
+wget https://github.com/aces/Loris/archive/v%VERSION%.zip
```

Comment:
Scripts use `$VARIABLE` not `%VARIABLE%` to refer to variables, I think it would make more sense to use that syntax for things that people are expected to type in in order to avoid confusion.